index.html

```diff
@@ -883,6 +941,18 @@ <h2>Determine if a performance entry buffer is full</h2>
         <li>Return true.</li>
       </ol>
     </section>
+    <section data-link-for="PerformanceEntry">
+      <h2>Generate a navigationId</h2>
+      <p>When asked to <dfn class="export">generate a navigationId</dfn> for a
+      <a>PerformanceEntry</a> <var>entry</a>, run the following steps:</p>
+      <ol>
+        <li>Let <var>type</var> be <var>entry</var>'s <a>entryType</a>.</li>
+        <li>Let <var>startTime</var> be <var>entry</var>'s <a>startTime</a>.
+        </li>
+        <li>Return the [=string/concatenation=] of « <var>type</var>,
```

Comment:
I know we initially talked about a UUID, but this may be equivalent. A few things that may hamper that:
* Can we have multiple navigations per ms? Unlikely, but I'm not sure it's currently prohibited.
* Having a predictable structure may cause developers to create weird reliances on the ID's structure (but I'm not sure that would be the case)
WDYT?

Reply:
- We can't have two navigations at the same monotonic time.
- I think it's OK that people rely on this if it's well defined. It's only the URL and the start time which are anyway available.

Reply:
Makes sense!

FWIW, when working on resource timing initiator (which uses a similar ID concept), we got [feedback](https://docs.google.com/document/d/1ODMUQP9ua-0plxe0XhDds6aPCe_paZS6Cz1h1wdYiKU/edit#heading=h.6m2qmrja1eyi) that it might be more ergonomic to use an object rather than a concatenated string.

Reply:
Do we run into problems with object equality, if we use a structured object? Specifically, are there issues around key order or numeric precision, or just the fact that object equality is often conflated with object identity?

Reply:
Discussed at the WG; there probably are issues with identity/equality here, and a string has better comparison behaviour, as long as we ensure it is unique.
Sticking with <type>-<timestamp> will allow this pattern to be reused for other cases where we want to refer to timeline entries as well.